apps/go-mod-validator/src/github.ts

```diff
@@ -24,104 +27,95 @@ async function isCommitInDefaultBranch(
 }
 
 async function isTagInDefaultBranch(
```

Comment:
This still doesn't deal with annotated tags right?

Reply:
It does, here's the output from the api call for `v2.15.0` from `chainlink`
``` json
{
    "name": "v2.15.0",
    "zipball_url": "https://api.github.com/repos/smartcontractkit/chainlink/zipball/refs/tags/v2.15.0",
    "tarball_url": "https://api.github.com/repos/smartcontractkit/chainlink/tarball/refs/tags/v2.15.0",
    "commit": {
      "sha": "883c9f9059c259ca17454703b57521ea9ac166cb",
      "url": "https://api.github.com/repos/smartcontractkit/chainlink/commits/883c9f9059c259ca17454703b57521ea9ac166cb"
    },
    "node_id": "MDM6UmVmMTExNDU1ODY3OnJlZnMvdGFncy92Mi4xNS4w"
  }
```
The tag's object ref is different
```
5f9c6900a76372940347436a4a0dbd713b710323 refs/tags/v2.15.0
```